src/components/Portal/Portal.tsx

```diff
@@ -43,7 +43,7 @@ export const Portal: FC<PortalProps> = ({onClose, hiddenOverflow, centered, disa
             theme === "light" ? "accent-color__backlog-blue" : "accent-color__planning-pink"
           )}
         >
-          <div className="portal__content" onClick={(e) => e.stopPropagation()} role="dialog">
+          <div className="portal__content" role="dialog">
```

Comment:
Why remove the stopPropagation here but insert it everywhere else?
Wouldn't this be the right spot in the first place to stop propagation?

Reply:
The portal__content is basically a big box around everything in the portal, i.e., in the stack view around the header, the parent note, and the scrolly bits at the bottom. Because of the new design, the scrolly part has to be the full width of the display, and because of the close button its height is the full height. If we have the `e.stopProgagation()` on the portal__content, clicking through this big box doesn't close the portal. In the case of the new stack view design, clicking the background to close the portal wouldn't work at all. (This also led to the stack view not being closable on mobile devices in the past).

With this new approach, it's also possible to click i.e., between cards in the stack view to close it, which I highly prefer.

Reply:
I think there still might be a better approach, but this will suffice for the moment :)